src/webapp/hooks/useMenu.ts

```diff
@@ -31,24 +33,28 @@ export function useMenu() {
     const [error, setError] = useState<string>();
     const snackbar = useSnackbar();
```

Comment:
Snackbar(render logic) outside the hook (presentation logic). It should be in the component and the hook should just send the message to display.

Reply:
done in https://github.com/EyeSeeTea/amr-surveys/pull/11